tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,21 @@
+k = 1

```

Comment:
If replace while with for you won't need k=1.

Reply:
removed